app/views/placements/providers/partner_schools/remove.html.erb

```diff
@@ -0,0 +1,25 @@
+<% content_for :page_title, t(".page_title", school_name: @partner_school.name) %>
+<%= render "placements/providers/primary_navigation", provider: @provider, current_navigation: :partner_schools %>
+<%= content_for(:before_content) do %>
+  <%= govuk_back_link(href: placements_provider_partner_school_path(@provider, @partner_school)) %>
+<% end %>
+
+<div class="govuk-width-container">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <label class="govuk-label govuk-label--l">
+        <span class="govuk-caption-l"><%= @partner_school.name %></span>
+        <%= t(".are_you_sure") %>
+      </label>
+      <%#= render GovukComponent::WarningTextComponent.new(
```

Comment:
Left this here

Reply:
Those are left there on purpose, as a remind to add the warnings when doing the second part
https://github.com/DFE-Digital/itt-mentor-services/pull/452

Reply:
Actually given I've now done the second part, I'll remove the comments

Reply:
resolved in [9c174f8](https://github.com/DFE-Digital/itt-mentor-services/pull/449/commits/9c174f8926178e92f3eab11b1000174939c20150)